Kickstarter-iOS/DataSources/ProjectNavigatorPagesDataSource.swift

```diff
@@ -18,6 +18,10 @@ internal final class ProjectNavigatorPagesDataSource: NSObject, UIPageViewContro
     self.padControllers(toLength: self.playlist.count)
   }
 
+  internal func updatePlaylist(_ playlist: [Project]) {
+    self.playlist = playlist
```

Comment:
This is almost too simple to be real, but I haven't seen any issues. Basically just replacing the UIPageViewController list of projects completely. Is it really this simple?